person.rb

```diff
@@ -20,6 +21,11 @@ def correct_name
     @name
   end
 
+  def add_rental(rental)
+    @rentals.push(rental)
+    rental.person = self
+  end
```

Comment:
- [ ] Let's modify this method as we did earlier. Since we are in the person class we have access to the person through `self`. You just need to accept `date` and `book` parameters and create the rental inside. 🧐️

```
def add_rental(date, book)
  Rental.new(date, book, self)
end
```